pkg/wipeout/wipeout.go

```diff
@@ -0,0 +1,204 @@
+// Copyright 2022 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package wipeout
+
+import (
+	"context"
+
+	"github.com/apigee/registry/cmd/registry/core"
+	"github.com/apigee/registry/connection"
+	"github.com/apigee/registry/log"
+	"github.com/apigee/registry/rpc"
+)
+
+// Set this global variable in calling code to run verbosely.
+var Verbose bool
```

Comment:
Have we considered using flags or some other method for this config? I think this is the first use of config by setting a global var.

Also worth considering whether we need this at all, especially in the "minimal" initial benchmarks. We already have multiple logging levels that give control over the verbosity of output logs. If we want wipeout to be silent by default and optionally turned on, maybe we should just log at debug level instead of adding another dial for controlling log output.

Reply:
Thanks, I'll use a followup PR to change this to logging at Debug. How would we set that for a specific run of the test? (directly in code?)

Reply:
Done in https://github.com/apigee/registry/pull/639/commits/19f552bb3e9abe726a6151dc3243eb9e7d50ef4a